pom.xml

```diff
@@ -640,15 +703,6 @@
                                         <exclude>org/apache/hadoop/util/LineReader.class</exclude>
                                         <exclude>org/apache/hadoop/crypto/key/kms/KMSClientProvider.class</exclude>
                                         <exclude>org/apache/hadoop/crypto/key/kms/KMSClientProvider$*.class</exclude>
-                                        <exclude>org/apache/hadoop/security/LdapGroupsMapping.class</exclude>
-                                        <exclude>org/apache/hadoop/security/LdapGroupsMapping$*.class</exclude>
-                                    </excludes>
-                                </filter>
-                                <filter>
-                                    <artifact>org.apache.hadoop:hadoop-auth</artifact>
-                                    <excludes>
-                                        <exclude>org/apache/hadoop/security/authentication/util/KerberosUtil.class</exclude>
-                                        <exclude>org/apache/hadoop/security/authentication/util/KerberosUtil$*.class</exclude>
```

Comment:
I think the subsequent filter for `org.apache.hadoop:hadoop-azure` needs update. I got a class not found when running `TestDeltaLakeAdlsConnectorSmokeTest` locally.

Reply:
I have updated the filter for `org.apache.hadoop:hadoop-azure`.
However, I can't run `TestDeltaLakeAdlsConnectorSmokeTest` because I don't have access to azure-abfs.

https://github.com/trinodb/trino/blob/master/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsConnectorSmokeTest.java#L64-L66